fiat-roles/src/main/java/com/netflix/spinnaker/fiat/providers/internal/ClouddriverService.java

```diff
@@ -25,4 +26,7 @@
 
   @GET("/credentials")
   List<Account> getAccounts();
+
+  @GET("/applications?restricted=false&expand=false")
```

Comment:
I don't have any reason (and don't forsee any need in the future) to call them with different parameters, otherwise I would.